basic-examples/expressions/README.md

```diff
@@ -2,16 +2,78 @@
 
 ## Workflow Description
 
-This workflow demonstrates example expressions as detailed in our [expression documentation](https://arcalot.io/arcaflow/workflows/expressions/).
+This workflow demonstrates some example expressions as detailed in our [expression documentation](https://arcalot.io/arcaflow/workflows/expressions/). Arcaflow expressions are a way to reference paths within the workflow or their fields, and/or manipulate the data.
+
+Arcaflow expressions were inspired by JSONPath but have diverged from the syntax. You can use expressions in a workflow YAML like this:
+

```

Comment:
Is the history lesson really useful here?  It sounds like it's not...so I would omit it and just go with the example.

Reply:
If the message you are trying to send is, "this is not JSONPath", I think you've buried the lead.